app/components/Instructions/Instructions.js

```diff
@@ -2,57 +2,55 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 export default function Instructions({ state, }) {
-  function buildInstructionArray(appState) {
+  function createInstructionArray(appState) {
     const instructionList = [];
     const {
       pathStartingPoint,
       pathEndingPoint,
       startButtonPressed,
       endButtonPressed,
     } = appState;
-    const setGoBackText = buttonText => (
+    const getGoBackText = buttonText => (
       `Press on the "${buttonText}" button in order to go back to toggle mode`
     );
-    const setReferencePoint = buttonText => (
+    const getReferencePoint = buttonText => (
```

Comment:
The line above ends with "Text", but this one doesn't. Be consistent. Decide how to name all similar parts and follow your decision.

Reply:
done.
Thought adding the word `Text` at the end of the variables below. 
Then felt it would be a useless repetition.

Now, if I could put them all inside an object called `instructions`, but then I would destructure it and all the magic of using `instructions['toggleTiles']` would go away. Is there sort of a middle ground?

Reply:
instructionTexts?

Reply:
But what about the retrieval of the strings? Wouldn't it be problematic with the DRY principle that I would use ```instructionTexts``` several times?

Reply:
Destructure it?